tests/fuzz/fuzz-bip32.c

```diff
@@ -0,0 +1,52 @@
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+#include <tests/fuzz/libfuzz.h>
+
+#include <ccan/tal/tal.h>
+#include <common/bip32.h>
+#include <wally_bip32.h>
+
+void init(int *argc, char ***argv)
+{
+}
+
+void run(const uint8_t *data, size_t size)
+{
+	struct ext_key xkey;
+	struct bip32_key_version version;
+	uint8_t *wire_buff;
```

Comment:
why eschew the short_types??

Reply:
We were already using it for the function's signature so..